client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/sidebar.tsx

```diff
@@ -23,9 +27,13 @@ const Sidebar = ( { siteSlug }: { siteSlug: string | null } ) => {
 					</div>
 				</div>
 				{ /* eslint-disable-next-line wpcalypso/jsx-classname-namespace*/ }
-				<h1 className="launchpad__sidebar-h1">Voilà! Your Newsletter is up and running!</h1>
+				<h1 className="launchpad__sidebar-h1">
+					{ translate( 'Voilà! Your %(translatedFlowName)s is up and running!', {
```

Comment:
Just noting that adding descriptive comments above translation strings can give translators more context and help them out a bit.